explore/src/main/scala/explore/ExploreLayout.scala

```diff
@@ -68,6 +75,9 @@ object ExploreLayout {
                 props.r.renderP(props.view)
               )
             )
+          )(
+            ^.onClick -->
+              helpView.set(none).runAsyncAndForgetCB.when_($.state.helpVisible)
```

Comment:
What happens if we always set `none` here without tracking the status in the state?

Reply:
Actually we should be able to simplify. I got a bit confused by this one getting called twice but it was avoided by preventing propagation on the help icon